README.md

```diff
@@ -71,14 +75,23 @@ scenarios that are supported by Chariott. It can be found
 
 ## Getting started
 
+### Important Note
```

Comment:
This section should be called **Requirements** and go before **Getting Started**.

Reply:
Done in https://github.com/eclipse/chariott/pull/45/commits/771c33ec91fa9fd935cf97e32c61f6eb82d8e1df